### DISCUSSION TITLE
Moved project to another region... how do I redeploy cloud functions

### DISCUSSION THREAD

**Discussion by alikibao** [AntlerVC/firetable#1153] [Support / Q&A] [answered]
Hi All

I duplicated a firebase project to another region. All that data but of course under a different project name. 

Rowy however, did not treat this as a new project because all the Rowy documents were also imported to the new project. Whatever I deploy now sits at "Build pending..."

Any ideas on how I can sort this out? Thanks

**Comment by alikibao** [comment, marked as answer]
I think I got it right... Please let me know if I missed anything

1. Removed project from Rowy
2. I removed the _row_/settings collection in the new project then reconnected Rowy, this way it prompted me to reinstall Rowy Run
3. Manually copied the contents of _rowy_/settings/functions, /_rowy_/settings/schema from the old to new project
4. Manually added the "tables" array to _rowy_/settings from the old project

After the above, all the tables and functions for the old project showed up in Rowy, and then all I had to do was go to each table and redeploy functions. 

Whether all functions work as expected remains to be seen.

**Reply by bofeiw** [reply]
You are correct! However table functions will need to be re-deployed manually.
